app/src/main/java/org/wikipedia/page/PageProperties.java

```diff
@@ -65,9 +65,7 @@ public PageProperties(@NonNull PageSummary pageSummary) {
         leadImageWidth = pageSummary.getThumbnailWidth();
         leadImageHeight = pageSummary.getThumbnailHeight();
         String lastModifiedText = pageSummary.getTimestamp();
-        if (lastModifiedText != null) {
-            lastModified.setTime(iso8601DateParse(lastModifiedText).getTime());
-        }
+        lastModified.setTime(iso8601DateParse(lastModifiedText).getTime());
```

Comment:
Should we have a `isNotEmpty()` check on this line since it might be an empty value?